ui/src/plugins/epl-ets/components/TableView.js

```diff
@@ -0,0 +1,285 @@
+import 'react-virtualized/styles.css';
+import './Table.css';
+
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
+
+class TableView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sortBy: '',
+      sortDirection: SortDirection.ASC,
+      selectLabel: 'Select columns to show',
+      selectOptions: [
+        { value: 'name', label: 'name' },
+        { value: 'size', label: 'Size' },
+        { value: 'memory', label: 'Memory' },
+        { value: 'type', label: 'Type' },
+        { value: 'write_concurrency', label: 'Write concurrency' },
+        { value: 'read_concurrency', label: 'Read concurrency' },
+        { value: 'lookup_max', label: 'lookup max time' },
+        { value: 'lookup_count', label: 'lookup count' },
+        { value: 'insert_max', label: 'insert max time' },
+        { value: 'insert_count', label: 'insert count' },
+        { value: 'owner', label: 'Owner' },
+        { value: 'heir', label: 'Heir' },
+        { value: 'protection', label: 'Protection' }
+      ],
+      selectedOptions: [
+        { value: 'name', label: 'Name' },
+        { value: 'size', label: 'Size' },
+        { value: 'memory', label: 'Memory' },
+        { value: 'type', label: 'Type' },
+        { value: 'write_concurrency', label: 'Write concurrency' },
+        { value: 'read_concurrency', label: 'Read concurrency' },
+        { value: 'lookup_max', label: 'lookup max time' },
+        { value: 'lookup_count', label: 'lookup count' },
+        { value: 'insert_max', label: 'insert max time' },
+        { value: 'insert_count', label: 'insert count' }
+      ]
+    };
+  }
+
+  is_selected(val) {
+    var listOfObjects = this.state.selectedOptions.filter(({ value, ...r }) => {
+      return value === val;
+    });
+    if (listOfObjects.length > 0) {
+      return true;
+    }
+    return false;
+  }
+
+  propComparatorASC(prop) {
+    return function(a, b) {
+      return a[prop] - b[prop];
+    };
+  }
+
+  propComparatorDESC(prop) {
+    return function(a, b) {
+      return b[prop] - a[prop];
+    };
+  }
+
+  sort(listOfObjects) {
+    if (this.state.sortDirection === SortDirection.ASC) {
+      return listOfObjects.sort(this.propComparatorASC(this.state.sortBy));
+    } else {
+      return listOfObjects.sort(this.propComparatorDESC(this.state.sortBy));
+    }
+  }
+
+  render() {
+    var tabData = this.props.table.tabs.filter(function(node) {
+      var newName = this.split('.').join('_').replace('@', '_at_');
+      return node.name === newName;
+    }, this.props.table.node);
+    if (tabData[0].length < 1) return null;
+    const list = tabData[0].tabs.map(function({ info, call_stats, ...a }) {
+      var callStatsLookup = {
+        lookupMax: 0,
+        lookupCount: 0
+      };
+      var callStatsInsert = {
+        insertMax: 0,
+        insertCount: 0
+      };
+      if (call_stats !== 'undefined' && typeof call_stats !== 'undefined') {
```

Comment:
Try to log out `call_stats` and see that it's equal to. If there is no property in the object it should be `undefined` and `if (call_stats)` should work. https://repl.it/Jm7s